src/components/module_parser.js

```diff
@@ -11,22 +11,36 @@ const updateModuleTypeList = async (oldModuleTypeList, jsonFilePath) => {
 
 const ModuleParser = {
 
+    /**
+     * Check if a module is an info module
+     * Info modules contain INFO in their name
+     */
+    isNotInfoSemester: (hsluModuleName) => {
+        return !hsluModuleName.includes('INFO')
+    },
```

Comment:
When a method returns a `bool`, always think of what it should check, not what it shouldn't. This makes the code more readable.

```suggestion
    isInfoModule: (hsluModuleName) => {
        return hsluModuleName.includes('INFO')
    },
```